### PR TITLE
Added reference to Office Add-Ins resources

### PR DESCRIPTION
**Pull request by LezaMax**
Included a pointer to samples and get started specific for Office Add-Ins.

### CODE REVIEW

File: ghdocs/GETTINGSTARTED.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ The following starter template represents the minimum recommended HTML structure
 ##Samples
 
 To see several examples of Fabric in use, check out the sample HTML projects under `dist/samples`. The following samples are included:
-
+- **Office Add-in**. A simple Office add-in that showcases the use of Fabric and some of its components. This sample has a [project on its own.](https://github.com/OfficeDev/Office-Add-in-Fabric-UI-Sample).
```
Comment (Jahnp) [inline ghdocs/GETTINGSTARTED.md]: Does add-in need to be capitalized?

**Comment by Jahnp** [comment]
Looks great!

File: ghdocs/GETTINGSTARTED.md (hunk 1 above)
Comment (LezaMax) [inline ghdocs/GETTINGSTARTED.md]: I think is fine if its all lower case

Comment (Jahnp) [inline ghdocs/GETTINGSTARTED.md]: Cool, then I'll merge this in. Thanks!

**Pull request merged**